### ISSUE TITLE
High Contrast themes have the same border color for focused and unfocused active tabs

### ISSUE THREAD

**Issue by Z-E-D** [microsoft/vscode#145563]
High Contrast themes have "focusBorder" color that is the same for the active and unfocusedActive tabs, i.e. the active tabs in different groups:

![Tabs](https://user-images.githubusercontent.com/97363660/159281818-28b14dba-7424-4e94-871b-e44b21ab56b5.png)
:

Because of that it is somewhat harder to distinguish which group currently has the focus. It would be better if the border line of the active tab in unfocused group is dotted (to be different than the hovered tabs that have dashed border line). Or, even better, it would be nice if you add one new color setting available to themes, e.g. "unfocusBorder", or something like that.

**Comment by daviddossett** [comment]
@bpasero I see we're drawing the outline for any active tab in a high contrast theme. It doesn't look like we're checking for the active editor group if I'm reading the code correctly:

https://github.com/microsoft/vscode/blob/4b9253809f3fd3bc21c19cd5e19e02d211656e66/src/vs/workbench/browser/parts/editor/tabsTitleControl.ts#L1906-L1914

We already used the dashed border for hover on inactive tabs, which I think makes sense. One thought here would be use a less prominent color that somehow conveys "inactive" without dimming the text like we do for many other normal light/dark themes. This is just using an unrelated token to get the point across:

<img width="566" alt="CleanShot 2022-03-30 at 14 50 53@2x" src="https://user-images.githubusercontent.com/25163139/160936988-9042d84a-81c1-4ef0-8168-b8899895e54f.png">

I'd agree that it would be useful to have this theme token exposed for themes to configure.

**Comment by bpasero** [comment]
💯 for using a different contrast color for active vs. inactive group 👍

**Comment by Z-E-D** [comment]
I am glad that you are taking this onto the consideration.

I just want to mention again my suggestion with the dotted line, because I think it has its analogy with the items in the primary side bar. For example, if you have displayed Open Editors and Explorer at the same time; there we have a solid line for active item in focused panel (e.g. Open Editors), a dashed line for hovered item and a dotted line for the active item in unfocused panel (e.g. Explorer).

**Comment by Z-E-D** [comment]
Now when I think more about this issue, I want to add one more suggestion, if you don't mind. If I have e.g. opened two groups,  I think that both active tabs in these two groups should have dotted lines when they both lose focus, e.g. if user activate side panel or Terminal or whatever else.

Here is again the analogy with the side bar. If I have opened two panels, e.g. Open Editors and Folders, and if one of them is focused, its active item will have solid line while the active item in the unfocused panel will have dotted line. However, if you give focus to editor, they both (Open Editors and Folders) will have active items with the dotted line, because neither of them has focus anymore.

I think that the same thing should be applied to the border line of the tabs, i.e. none tab should have solid border line when none editor has focus.
